dockerfiles/hpcc/templates/dllserver-claim.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     - ReadWriteMany
```

Comment:
This is for local hostPath or NFS type, right?  For dynamic EBS or similar we probably should use StatefullSet instead which probably will be hard to make both "Deployment" and "StatefulSet" implementation in one Helm/Charts

Reply:
At the moment I'm only trying to support local hostPath.

I suspect for other types of location we can make entirely different dllserver/dllserver-claim templates.